pkg/controllers/deprovisioning/emptymachineconsolidation.go

```diff
@@ -82,7 +82,8 @@ func (c *EmptyMachineConsolidation) ComputeCommand(ctx context.Context, candidat
 	// the machine isn't a target of a recent scheduling simulation
```

Comment:
I wonder if it makes more sense to create an `ErrValidationFailed` error type and then log about this validation failure generally